assets/kube-state-metrics/kube-state-metrics-deployment.yaml

```diff
@@ -78,13 +78,7 @@ spec:
         image: quay.io/coreos/kube-state-metrics:v1.3.1
         imagePullPolicy: IfNotPresent
         name: kube-state-metrics
-        resources:
-          limits:
-            cpu: 102m
-            memory: 180Mi
-          requests:
-            cpu: 102m
-            memory: 180Mi
```

Comment:
We should also remove the add on resizer here, I think it will also require changes in the kube state metrics task in pkg/tasks as there is some manual “merge” logic there because of the addon-resizer.